velox/exec/Operator.h

```diff
@@ -555,7 +555,7 @@ class Operator : public BaseRuntimeStatWriter {
 
   /// Creates output vector from 'input_' and 'results_' according to
   /// 'identityProjections_' and 'resultProjections_'.
-  RowVectorPtr fillOutput(vector_size_t size, BufferPtr mapping);
+  RowVectorPtr fillOutput(vector_size_t size, const BufferPtr& mapping);
```

Comment:
It would be nice to update the comment to explain how 'mapping' is used and clarify that it can be nullptr.

Reply:
Updated, thanks.